canonical-web-and-design/maas-ui
# Build zones table

## Done

- Add sortable table with links to machines with tests
- Add counter to header

## QA

- run `yarn start` and go to http://0.0.0.0:8400/MAAS/r/zones
- Check the table is visible and the links are correct

![Screenshot 2021-06-08 at 14 02 45](https://user-images.githubusercontent.com/58959073/121189693-3c5cd880-c862-11eb-86c8-fb36ad212d76.png)
